src/probability_theory/stopping.lean

```diff
@@ -0,0 +1,294 @@
+/-
+Copyright (c) 2021 Kexing Ying. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kexing Ying
+-/
+import probability_theory.notation
```

Comment:
Do you use any of the notations defined in `probability_theory.notation`? If no, could you replace that import by some other file(s)?
It looks like you don't actually need much more than `measure_space` (or even `measruable_space`?).

Reply:
I've minimized it now, though I will probably need to add more imports as I work on the file